templates/automotive/index.html

```diff
@@ -220,43 +65,348 @@ <h2 class="p-muted-heading u-align--center u-sv3">THESE AUTOMOTIVE INNOVATORS HA
-          Get in touch today
-        </a>
-      </p>
+      <h2 class="p-heading--five">Industry 4.0</h2>
+      <p class="p-heading--two">Faster factories, smarter robots</p>
+      <p>Digital transformation and continuous improvement with app-centric factory automation.</p>
+      <p>Monitor inventories. Spot defects. Manage warehouses. Optimise supply lines.</p>
+      <p>Run simulations for robot training and deploy new insights automatically in a snap. No downtime.</p>
     </div>
-    <div class="col-6 u-hide--small u-align--center u-vertically-center">
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
       {{
         image(
-          url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
+          url="https://assets.ubuntu.com/v1/22007291-DevOps.svg",
           alt="",
-          height="178",
-          width="250",
+          height="200",
+          width="350",
           hi_def=True,
-          loading="lazy",
         ) | safe
       }}
     </div>
   </div>
 </section>
 
+<section class="p-strip">
+  <div class="row u-equal-height">
+    <div class="col-6 u-vertically-center u-align-center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/0631c9dc-FinancialServicesPage-GetStarted-AI_2020.svg",
+          alt="",
+          height="200",
+          width="219",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+    <div class="col-6">
+      <h2 class="p-heading--five">Deep learning</h2>
+      <p class="p-heading--two">Accelerate data science &ndash; car, cluster and cloud</p>
+      <p>From self-driving to diagnostics and route planning, Ubuntu is at the heart of the biggest automotive analytics operations.</p>
+      <p>More frameworks, languages, libraries and solutions, from the widest range of companies and clouds.</p>
+      <p>On <a href="https://maas.io/" class="p-link--external">bare metal</a> or <a href="/cloud">public cloud</a>. From Spark and Hadoop to <a href="/kubeflow">Kubeflow</a>. Better ML. Faster results.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h2 class="p-heading--five">Scania presents</h2>
+      <p class="p-heading--two">Fast track CAD/CAM/CAE with dynamic HPC</p>
+      <p>Scania deploys agile HPC clusters for rapid digital design, validation, crash testing, and aerodynamics.</p>
+      <p>Supercomputers on-demand for CAD, CAM and CAE. Tame SLURM and get more from your cluster.</p>
+      <p><a href="https://www.brighttalk.com/webcast/6793/416900/mastering-multicloud-with-slurm-and-juju" class="p-link--external">Watch the video&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/1c2a3c27-Automotive-Masters-vid-screen.png",
```

Comment:
this is meant to be a video?

Reply:
It's linked to an image in the copy doc but I'll try add a link around the image